spec/Section 5 -- Validation.md

```diff
@@ -1131,8 +1131,8 @@ fragment catInDogFragmentInvalid on Dog {
 
 ##### Abstract Spreads in Object Scope
 
-In scope of an object type, unions or interface spreads can be used
-if the object type implements the interface or is a member of the union.
+In scope of an object or interface type, unions or interface spreads can be
```

Comment:
Rather than changing this, I'd suggest a new section. This section is about what spreads you may use in the context of an Object type.

The section after this is about what spreads you may use in the context of an Abstract type.

Perhaps there should be one more section about `Abstract Spreads in Abstract Scope`?

Reply:
This looks like a mistake of mine. I removed the "or interface", and applicable appropriate language already exists under "Abstract Spreads in Abstract Scope":

> Union or interfaces fragments can be used within each other. As long as there exists at least *one* object type that exists in the intersection of the possible types of the scope and the spread, the spread is considered valid.